django/db/migrations/operations/models.py

```diff
@@ -43,11 +43,12 @@ class CreateModel(ModelOperation):
 
     serialization_expand_args = ['fields', 'options', 'managers']
 
-    def __init__(self, name, fields, options=None, bases=None, managers=None):
+    def __init__(self, name, fields, options=None, bases=None, managers=None, metaclass=None):
```

Comment:
This function is [documented](https://docs.djangoproject.com/en/3.1/ref/migration-operations/). Please update the docs accordingly. Also add a `..versionchanged ::3.2` annotation. 

https://docs.djangoproject.com/en/3.1/internals/contributing/writing-code/submitting-patches/#new-features

Also as this is a change to a feature, please add it to the release notes. `docs/releases/3.2.txt`

Reply:
@smithdc1 it seems like only the docs are failing because of spelling? I ran the tests locally and it didn't show a problem there. Do you have any ideas?